src/fields/CheckBox.jsx

```diff
@@ -14,7 +14,7 @@ export default class CheckBox extends Component {
     return (
       <div>
         <label className={labelClass} ref="label">
-          <input {...cleanProps(props)} type="checkbox" ref="box"/>
+          <input {...cleanProps(props)} type="checkbox" ref="box" checked={props.value === true}/>
```

Comment:
This is interesting and potentially expectation-breaking behaviour. I understand why this makes sense but strictly speaking a "value" and a "checked state" are different things. Passing in `checked: true` for a checkbox field should already achieve this (unless `cleanProps()` removes it?)